test/ui/SimpleFilterTest.ts

```diff
@@ -0,0 +1,181 @@
+import * as Mock from '../MockEnvironment';
+import { ISimpleFilterOptions, SimpleFilter } from '../../src/ui/SimpleFilter/SimpleFilter';
+import { Simulate } from '../Simulate';
+import { $$ } from '../../src/utils/Dom';
+import { BreadcrumbEvents, IPopulateBreadcrumbEventArgs } from '../../src/events/BreadcrumbEvents';
+import { FakeResults } from '../Fake';
+
+export function SimpleFilterTest() {
+  describe('SimpleFilter', () => {
+    let test: Mock.IBasicComponentSetup<SimpleFilter>;
+    let test2: Mock.IBasicComponentSetup<SimpleFilter>;
+    let test3: Mock.IBasicComponentSetup<SimpleFilter>;
+
+    beforeEach(() => {
+      test = Mock.optionsComponentSetup<SimpleFilter, ISimpleFilterOptions>(SimpleFilter, <ISimpleFilterOptions>{
```

Comment:
I need at least two for some tests (making sure there is only one backdrop, etc). The third one is used for the 2nd code path (groupBy values). I could remove the third one by setting the first one's values to [] before using it.